near-sdk-macros/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "near-sdk-macros"
-version = "4.1.1"
+version = "4.1.2"
```

Comment:
Please, revert version bumps.

1. Releases are cut in a separate PR
2. This change is a breaking change for 4.x, so we will need to consider it for 5.x and in 4.x we need to only deprecate the macros (deprecation can be a patch release as per semver, but removal cannot)

Reply:
Yeah, I had a suspicion about this, that's why all the version changes are separated into commits we can revert.